### ISSUE TITLE
success:disown:1: no current job

### ISSUE THREAD

**Issue by quiteDEADlee** [sudofox/melee.sh#5]
Getting an error on my MacBook Pro M1 under zsh in iTerm2:
`success:disown:1: no current job`

**Comment by sudofox** [comment]
PR #4 addresses this however I've asked for additional improvements before I can accept it (too bulky)

**Comment by Cyber-Cowboy** [comment]
Also got similar problem when I was trying to run one of examples, there was no sound and this message at the end:
`failure:disown:2: no current job`
I'm on ubuntu with zsh.

**Comment by sudofox** [comment]
zsh is not currently supported per the aforementioned thingy

(I've nothing against zsh it's just that I use bash)

Maybe I'll try zsh for a bit to see if I can implement it myself

**Comment by Cyber-Cowboy** [comment]
Yeah, no problem, thanks for a good tool I currently using it with PR #4 except I replaced the code in  current_shell() so it always returns zsh, not an optimal solution but it works for me.
